samples/deepseek-r1-webgpu/src/App.jsx

```diff
@@ -187,7 +187,7 @@ function App() {
               href="${relativePath}/index.html"
               class="font-mono font-bold text-4xl text-stone-50 text-shadow-sm block px-8 xl:px-16"
               >
-                  <svg class="2xl:w-[287px] w-[180px]" viewBox="0 0 887 157" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <svg class="w-[180px] 2xl:w-[220px]" viewBox="0 0 887 157" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
https://github.com/intel/web-ai-showcase/blob/main/js/navigation.js#L16 define the header navigator for main and other demo pages, also https://github.com/intel/web-ai-showcase/blob/main/samples/phi3-webgpu/src/App.jsx#L240 needs override to make main home page and sub demos pages unified.

Reply:
Thanks a lot! Fixed